src/app/page.tsx

```diff
@@ -1,14 +1,106 @@
-import React from "react";
+"use client"
 
+import React, { useEffect, useState } from "react";
+import { SupplierCard } from '@/components/mols';
+import { CategoryCard } from '@/components/atoms'
+import SupLogo from '@/images/easytolive/logo/logotipo-fundoazulroxo.svg'
+import SearchCategory from '@/app/searchCategory'
+import supplierService from '@/service/supplier.service'
+import imageCategory from '@/images/easytolive/icons/categorie-example.svg'
+import { ISupplier, ISupplierList } from '@/types/supplier';
+import { useDebounce } from 'use-debounce';
+import { categorieProps } from '@/components/atoms/CategoryCard';
+import InfiniteScroll from 'react-infinite-scroll-component';
 
 function PageHome() {
+  const [suppliers , setSuppliers] = useState([])
+  const [search, setSearch] = useState('');
+  const [textSearched] = useDebounce(search, 1000);
+  const [categorys, setCategorys] = useState([]);
+  const [pageNumber, setPageNumber] = useState(1);
+  const [hasMore, setHasMore] = useState(true);
+  const [supplierCategoryFilter, setSupplierCategoryFilter] = useState('');
+
+  const getAllCategorys = async () => {
+    const res: any = await supplierService.getSupplierCategories();
+    return res;
+  }
+
+  useEffect(() => {
+    getAllCategorys()
+    .then((res)=> setCategorys(res?.data?.supplierCategories?.results))
+    .catch((error)=> console.log(error))
```

Comment:
Toast working now. We can show a toats here? 

If we receive a error to get all Categories?